src/main/java/io/numaproj/numaflow/mapper/MapperTestKit.java

```diff
@@ -149,6 +150,7 @@ public MessageList sendRequest(String[] keys, Datum data) {
                                     .setSeconds(data.getWatermark().getEpochSecond())
                                     .setNanos(data.getWatermark().getNano())
                                     .build())
+                    .putAllHeaders(data.getHeaders() == null ? new HashMap<>() : data.getHeaders())
```

Comment:
why is it getting initialized if null?

Reply:
proto builder functions throw an exception when we try to assign null values